x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/scalar/date/DateParseTests.java

```diff
@@ -103,7 +103,6 @@ public static Iterable<Object[]> parameters() {
         );
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/104124")
     public void testInvalidPattern() {
         String pattern = randomAlphaOfLength(10);
```

Comment:
How do you know this random pattern will be an invalid date format? This test seems still possibly flaky.

Reply:
Right, it's very unlikely but it's still possible that `randomAlphaOfLength(10)` returns a valid string pattern , and I don't have a better idea to randomize a string so that it's always invalid. I'll replace it with a constant string, it shoul be enough for our needs.